docs/Tutorials/uitoolkit.md

```diff
@@ -103,7 +103,7 @@ The code snippet below adds KVM control to the React application.
     | Field       |  Value   |
     | :----------- | :-------------- |
     | `deviceId` | Replace the example deviceId value with the GUID of the Intel® AMT device activated and connected to your MPS server. Information on obtaining a GUID can be found [here](../Topics/guids.md). |
-    | `mpsServer` | Replace the *localhost* with the IP Address of your Development Device or MPS Server. Default MPS Server port is 3000 or if using KONG, no port is used and should be `/mps/ws` |
+    | `mpsServer` | Replace the *localhost* with the IP Address of your Development Device or MPS Server. Default MPS Server port is 3000 or if using KONG, no port is used and should be `/mps/ws/relay` |
```

Comment:
I'm actually very confused by this group of sentences. The first part is great. IP Address of the Dev System and Port 3000. But the last part... I dunno. Is what I have below correct, or have I missed the meaning?

Replace the *localhost* with the IP Address of your Development Device or MPS Server. The default MPS Server port is 3000. If using KONG, no port number is necessary.  Append `/mps/ws/relay` to local host.

Reply:
``` suggestion
| `mpsServer` | Replace the *localhost* with the IP Address of your Development Device or MPS Server. The default MPS Server port is 3000. If using KONG, remove the port and append `/mps/ws/relay`  to localhost. |
```

Reply:
I like what Mike did there. It further disambiguated the instruction.